### PR TITLE
RPS-1 add solana client for basic operation

### PR DESCRIPTION
**Pull request by enslinmike**

### CODE REVIEW

File: solana/pom.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <artifactId>rarible-blockchain-scanner-solana</artifactId>
+    <version>${project.parent.version}</version>
+
+    <properties>
+        <ktor-client.version>1.6.6</ktor-client.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.ktor</groupId>
```
Comment (evgenynacu) [inline solana/pom.xml:21]: We use spring web client everywhere. Let's stick with it. I think no need to use new library

File: solana/src/main/kotlin/com/rarible/blockchain/solana/client/SolanaApi.kt [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        }
+    }
+
+    override suspend fun getLatestSlot(): Long = client.post<ApiResponse<Long>>(url) {
+        body = GetSlotRequest()
+    }.result
+
+    override fun getBlockFlow(): Flow<SolanaBlockchainBlock> = flow {
+        while (true) {
+            val slot = getLatestSlot()
```
Comment (evgenynacu) [inline solana/src/main/kotlin/com/rarible/blockchain/solana/client/SolanaApi.kt:50]: I think we can optimize here a little bit: do not try to load block if latest slot is the same as previous time

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: solana/pom.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            <artifactId>ktor-client-websockets</artifactId>
+            <version>${ktor-client.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.rarible.blockchain.scanner</groupId>
+            <artifactId>rarible-blockchain-scanner-core</artifactId>
+        </dependency>
+        <dependency>
```
Comment (yakischik) [inline solana/pom.xml:44]: Most of these dependencies already declared in core, check it please and remove re-declarations

**Review by yakischik — APPROVED** [review]

**Comment by yakischik** [comment]
We're working now in branch release/2.0, please re-create PR to it

**Pull request closed without merging**